pyrk/neutronics.py

```diff
@@ -120,19 +124,21 @@ def dwdt(self, power, omega, k):
         lam = self._dd.lambdas()[k]
         return kappa*p-lam*omega
 
-    def reactivity(self, t_idx, components):
+    def reactivity(self, t_idx, t_idx_feedback, components):
         """Returns the reactivity, in $\Delta k$, at time t
-        :param t: time
-        :type t: float, units of seconds
-        :param dt: timestep size, units of seconds
-        :type dt: float, units of seconds
+        :param t_idx: time step
+        :type t_idx: int, index
+        :param t_idx_feedback: time step
+        :type t_idx: int, index
         :param components: thermal hydraulic component objects
         :type components: list of THComponent objects
         """
         rho = {}
-        if self.feedback:
+        if self.feedback and t_idx > t_idx_feedback:
```

Comment:
no, because I use T(t_idx_feedback) as reference temperature for calculating the temperature difference for the reactivity feedback.